packages/dht/src/connection/websocket/WebsocketServerConnector.ts

```diff
@@ -75,7 +74,8 @@ export class WebsocketServerConnector {
         if (!this.abortController.signal.aborted && this.websocketServer) {
             this.websocketServer.on('connected', (connection: IConnection) => {
                 const serverSocket = connection as unknown as WebsocketServerConnection
-                const query = queryString.parse(serverSocket.resourceURL.query as string ?? '')
+                const urlParams = new URLSearchParams(serverSocket.resourceURL.query as string ?? '')
+                const query = Object.fromEntries(urlParams.entries())
```

Comment:
Does this code path get exercised in an existing test? The change looks innocent enough but just wondering if we had coverage for this.

Reply:
Good question. Took it from "as-is" from https://github.com/streamr-dev/network/commit/704b4e9d3682f45fdb58a87e4cbdcc2fb9ee9918. @ptesavol do you know by any chance?